R/build_multinom.R

```diff
@@ -0,0 +1,81 @@
+# build_multinom
+
+#' Build multinom model data frame
+#' @param data Data frame
+#' @param ... Parameters for nnet::multinom
+#' @export
+build_multinom <- function(data, ...){
+  build_model(data = data,
+              model_func = nnet::multinom,
+              reserved_colnames =  c(
+                # for model_coef
+                "y.level",
+                "term",
+                "estimate",
+                "std_error",
+                "t_ratio",
+                "p_value",
+                # for model_stats
+                "edf",
+                "deviance",
+                "AIC"
+              ),
+              ...)
+}
+
+#' augment for multinom model
```

Comment:
Please leave comment on that broom does not have augment for multinom, and we are adding that here.